src/App.css

```diff
@@ -0,0 +1,29 @@
+@font-face {
+  font-family: 'DistantGalaxy';
+  font-weight: 900;
+  src:
+    local('DistantGalaxy'),
+    url(./fonts/DistantGalaxy/SfDistantGalaxyAlternate-OE4d.ttf)
+      format('truetype');
+}
+
+#root {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  /* align-items: center; */
```

Comment:
remove

Reply:
These styles are responsible for footer being stuck to the bottom of the page, even when body doesn't have much content ..

Reply:
But if it was about the comment, I removed it!